### PR TITLE
Add_LibraryPass

### PR DESCRIPTION
**Pull request by Olive105**

### CODE REVIEW

**Comment by thatguyintech** [comment]
hey @albeethekid -- mind reviewing this PR?

File: src/cadence/mainnet/getNFTs.cdc [hunk 1]
```diff
@@ -262,8 +263,9 @@ pub fun main(ownerAddress: Address, ids: {String:[UInt64]}): [NFTData?] {
                 case "GoatedGoatsTrait": d = getGoatedGoatsTrait(owner: owner, id: id)
                 case "DropzToken": d = getDropzToken(owner: owner, id: id)
                 case "Necryptolis": d = getNecryptolisNFT(owner: owner, id: id)
+                case "LibraryPass": d = getLibraryPass(owner: owner, id: id)
                 case "FLOAT" : d = getFLOAT(owner: owner, id: id)
-                case "BreakingT_NFT": d = getBreakingTNFT(owner: owner, id: id)
+                case "BreakingT_NFT": d = (owner: owner, id: id)
```
Comment (prpatel05) [inline src/cadence/mainnet/getNFTs.cdc:268]: Looks like a method was removed.

File: src/cadence/testnet/getNFTs.cdc [hunk 2]
```diff
@@ -3194,4 +3196,39 @@ pub fun getByteNextMedalNFT(owner: PublicAccount, id: UInt64): NFTData? {
...
+
+    let col = owner.getCapability(LibraryPass.CollectionPublicPath)
+        .borrow<&{LibraryPass.CollectionPublic}>()
+
+    if col == nil { return nil }
+
+    let nft = col!.borrowLibraryPassNFT(id: id)
+    if nft == nil { return nil }
+
+    let metadata = Gaia.getTemplateMetaData(templateID: nft!.data.templateID)
```
Comment (prpatel05) [inline src/cadence/testnet/getNFTs.cdc:3221]: Same as above

**Review by prpatel05 — CHANGES_REQUESTED** [review]
I would recommend taking a look at https://github.com/alchemyplatform/alchemy-flow-contracts/blob/main/README.md and onboarding to the Catalog as well.  It will be a much simple and more self-service way to make your collection interoperable and discoverable.

File: src/cadence/mainnet/getNFTs.cdc (hunk 1 above)
Comment (Olive105) [reply src/cadence/mainnet/getNFTs.cdc:268]: I added a method

**Review by Olive105 — COMMENTED** [review]
I fixed the issue and pushed. Please review and let me know. Thanks

**Review by Olive105 — COMMENTED** [review]
issue - fix issue of metadata

**Comment by Olive105** [comment]
Hi Pratik, I removed the old pull request and created a new pull request.
Can you please review and merge asap so that there's no merge conflicts?
https://github.com/alchemyplatform/alchemy-flow-contracts/pull/171
Thanks.
Olive

**Pull request closed without merging**